From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Thu, 17 Aug 2023 02:16:28 +0200
Subject: ci: Use sudo to run `meson install`

Newer meson versions no longer use polkit for automatic priviledge
elevation.

Commit d353f2d367 adjusted to that change by running `meson install`
through sudo, but left out the install-meson-project helper script.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2897>
(cherry picked from commit bdbad8ca857d822bfe03dc9f14e2f7f2e6989271)
---
 .gitlab-ci/install-meson-project.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitlab-ci/install-meson-project.sh b/.gitlab-ci/install-meson-project.sh
index 8ecf8a3..6d1aa90 100755
--- a/.gitlab-ci/install-meson-project.sh
+++ b/.gitlab-ci/install-meson-project.sh
@@ -78,5 +78,5 @@ git clone --depth 1 "$REPO_URL" -b "$COMMIT" "$CHECKOUT_DIR"
 pushd "$CHECKOUT_DIR/$SUBDIR"
 sh -c "$PREPARE"
 meson setup --prefix=/usr _build "${MESON_OPTIONS[@]}"
-meson install -C _build
+sudo meson install -C _build
 popd
